### PR TITLE
Add method for mget with only ids and updated doc

### PR DESCRIPTION
**Pull request by triptec**
I want to use the short version of mget where you only pass index, type and ids like so:

```
curl 'localhost:9200/test/type/_mget' -d '{
    "ids" : ["1", "2"]
}'
```
https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-multi-get.html

Disclaimer:
I'm terribly new to golang and I bet I've broken some practises.

### CODE REVIEW

File: lib/coremget.go [hunk 1]
```diff
@@ -19,25 +19,49 @@ import (
...
+      "_type" : "type",
+      "_id" : "2"
+    }
+  ]
+}
+*/
 func (c *Conn) MGet(index string, _type string, mgetRequest MGetRequestContainer, args map[string]interface{}) (MGetResponseContainer, error) {
-	var url string
+	body, err := c.DoCommand("GET", mGetUrl(index, _type), args, mgetRequest)
+	return mGetHandleResponse(body, err)
```
Comment (weberr13) [inline lib/coremget.go]: Passing an error to a function is not really standard go.  I think it is more standard practice to just put the logic in here.

File: lib/coremget.go [hunk 2]
```diff
@@ -19,25 +19,49 @@ import (
...
+	"ids": ["1", "2", "3"]
+}
+*/
+
+func (c *Conn) MGetShort(index string, _type string, mgetRequestShort MGetRequestShort, args map[string]interface{}) (MGetResponseContainer, error) {
+	body, err := c.DoCommand("GET", mGetUrl(index, _type), args, mgetRequestShort)
+	return mGetHandleResponse(body, err)
+}
+
+func mGetHandleResponse (responseBody []byte, err error) (MGetResponseContainer, error) {
```
Comment (weberr13) [inline lib/coremget.go]: the mBlahBlah thing is not standard go and detracts from the readability of the code.  How about getHandleResponse?

Also, docs are helpful:

//getHandleResponse from the given response body

File: lib/coremget.go [hunk 3]
```diff
@@ -45,18 +69,34 @@ func (c *Conn) MGet(index string, _type string, mgetRequest MGetRequestContainer
 	return retval, err
 }
 
+func mGetUrl(index string, _type string) string {
```
Comment (weberr13) [inline lib/coremget.go]: same comment as the above.  Also, when you make these member functions it is a great time to write tests.  Please write tests.

**Comment by weberr13** [comment]
Please consider my suggestions and ping me when you update the pull request.

**Comment by triptec** [comment]
Good suggestions! Though I couldn't get the tests working, even on the master branch. Seems a few tests uses localhost even though I used the -host flag =(

Another time =)

**Pull request closed without merging**